src/Fixer/FunctionNotation/MethodArgumentSpaceFixer.php

```diff
@@ -61,6 +62,10 @@ public function getDefinition()
                     "<?php\nfunction sample(\$a=10,\$b=20,\$c=30) {}\nsample(1,  2);",
                     ['keep_multiple_spaces_after_comma' => true]
                 ),
+                new CodeSample(
+                    "<?php\nfunction sample(\$a=10,\n    \$b=20,\$c=30) {}\nsample(1,\n    2);",
+                    ['ensure_fully_multiline' => true]
```

Comment:
I added an example for the former, but the latter already documented, right? I mean since `false` is the default value for `keep_multiple_spaces_after_comma`... or do I need to explicitly add the option line 67?